src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -854,6 +854,20 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <PlatformTargetAsMSBuildArchitecture>x86</PlatformTargetAsMSBuildArchitecture>
     </PropertyGroup>
 
+    <Error Condition=" '$(Prefer32Bit)' == 'true' and '$(PreferNativeArm64)' == 'true' "
+           Code="MSB9901"
+           Text="Prefer32Bit and PreferNativeArm64 options are mutually exclusive. Please disable one of those."
```

Comment:
There are many examples of doing it this way in the targets, but it's bad to have unlocalized errors/warnings like these. Can this validation go into a task to get localized?

Reply:
The SDK made a repo-local task for emitting localized warnings or errors - I think that could be useful for this repo as well.